axelor-account/src/main/java/com/axelor/apps/account/service/ScaleServiceAccountImpl.java

```diff
@@ -70,7 +59,7 @@ public int getScale(Move move, boolean isCompanyAmount) {
   public int getScale(MoveLine moveLine, boolean isCompanyAmount) {
     return moveLine.getMove() != null
         ? this.getScale(moveLine.getMove(), isCompanyAmount)
-        : DEFAULT_SCALE;
+        : super.getScale();
```

Comment:
Here you can set `DEFAULT_SCALE` as protected in `ScaleServiceImpl` to have it on `ScaleServiceAccountImpl` as well
Also as this is an extension, you can directly call `this.getScale()`
Same thing for other similar cases in this class

Reply:
Resolved